spacy/tokens/doc.pyx

```diff
@@ -1486,6 +1486,15 @@ cdef class Doc:
                 token_data["dep"] = token.dep_
                 token_data["head"] = token.head.i
             data["tokens"].append(token_data)
+        
+        if self.spans:
+            data["spans"] = {}
+            for span_group in self.spans:
+                data["spans"][span_group] = []
+                for span in self.spans[span_group]:
+                    span_data = {"start": span.start_char, "end": span.end_char, "label": span.label_, "kb_id_": span.kb_id}
```

Comment:
```suggestion
                    span_data = {"start": span.start_char, "end": span.end_char, "label": span.label_, "kb_id": span.kb_id}
```

Reply:
> Would `kb_id_` be better here?


I'm a bit confused, did I change it in the wrong place? 😄

Reply:
Yes ;-) Adriane meant to suggest to query `span.kb_id_` just like we query `span.label_` and not `span.label`, but I guess the suggestion was a little ambiguous ;-)

Reply:
Aaahhh I didn't see that the suggested change included `span.kb_id_` so sorry! 😞